matplotlib_pyodide/browser_backend.py

```diff
@@ -190,6 +190,13 @@ def ignore(event):
 
         self.draw()
 
+    def destroy(self, *args, **kwargs):
+        div = document.getElementById(self._id)
+        parentElement = div.parentNode
+        if parentElement:
+            parentElement.removeChild(div)
+        div.removeChild(div.firstChild)
```

Comment:
Question:

The `parentElement` removed the child `div`, then can `div` survive here?

Reply:
No. The parent of child of the div is the div itself, so this code means deleting itself.